copter/source/docs/traditional-helicopter-internal-rsc-governor.rst

```diff
@@ -4,29 +4,43 @@
 
-With auto landings or the LAND mode, upon detecting landing, the system will pull the engine's throttle down to flight idle power (:ref:`H_RSC_IDLE<H_RSC_IDLE>`), disengaging the governor before auto-disarm and engine shutdown. This reduces the chances of a dynamic rollover vs governors that hold full headspeed while the system is on the ground.
+Once the throttle curve is tuned, set :ref:`H_RSC_MODE<H_RSC_MODE>` = 4 to have the RSC use the governor. Initially it is recommended that the governor be tuned for the RPM that the throttle curve was tuned.  This will minimize the need for the governor torque compensator to be used and allow tuning to be concentrated on the droop compensator and feedforward settings.  The feedforward setting adjusts the amount of the throttle curve is used in the governor.  The feedforward may need adjusting if the rotor speed droops significantly when loading the rotor system using the collective.  The droop compensator ensures the desired rotor speed is maintained very precisely.  Higher value is quicker response to large speed changes due to load but may cause surging. Adjust this to be as aggressive as possible without getting surging or RPM over-run when the governor responds to large load changes on the rotor system.
 
-If you desire the governor to never disengage once it engages, set the ``H_RSC_GOV_DISGAG`` to zero and it won’t reduce the throttle no matter what you do, unless Motor Interlock is disengaged.
+To investigate the effect of the torque compensator, set the desired rotor speed 100-200 RPM from the rotor speed the throttle curve was tuned.  The governor torque compensator (:ref:`H_RSC_GOV_COMP<H_RSC_GOV_COMP>`) should slowly adjust the reference output to aid the droop compensator in maintaining the desired RPM.  If RPM is low or high by more than 2-5 RPM, increase this setting by 1% at a time until the governor speed matches your RPM setting. Setting the compensator too high can result in surging and throttle "hunting". Do not make large adjustments at one time.  
 
-You can refer to the :ref:`ArduPilot Helicopter Ground School videos<traditional-helicopter-configuration>` if you want to take a refresher on setting throttle curves. Video 4 covers throttle curves, piston and turbine engines:
+The torque limiter (:ref:`H_RSC_GOV_TORQUE<H_RSC_GOV_TORQUE>`) should not need to be adjusted unless desired rotor speeds are set above the rotor speed tuned for the throttle curve. The governor failing to engage on spool up is an indication that the torque limiter is too low.  Raise this be 5% until the governor reliably engages.  It is possible to have this too high which would be indicated by immediately getting a governor overspeed fault on spool up.
```

Comment:
need to explain what that fault mainfests as

Reply:
meant how does the user know there is an overspeed fault....gcs message?

Reply:
Yes there is a GCS message and it defaults to the throttle curve.  I will fix that tonight.

Reply:
fixed